lib/cartopy/mpl/patch.py

```diff
@@ -64,7 +65,9 @@ def geos_to_path(shape):
             paths.extend(geos_to_path(shp))
         return paths
 
-    if isinstance(shape, (sgeom.LineString, sgeom.Point)):
+    if isinstance(shape, sgeom.LinearRing):
+        return [Path(np.column_stack(shape.xy), closed=True)]
+    elif isinstance(shape, (sgeom.LineString, sgeom.Point)):
```

Comment:
Note: a `LinearRing` is a `LineString`, so that's why this conversion worked before.